src/main/java/edu/wpi/grip/ui/ExceptionView.java

```diff
@@ -0,0 +1,191 @@
+package edu.wpi.grip.ui;
+
+import javafx.application.HostServices;
+import javafx.application.Platform;
+import javafx.event.ActionEvent;
+import javafx.scene.control.*;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Priority;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Displays an alert with an exception and provides a place for a user to supply information about what caused the error.
+ * Includes a textbox with formatted markdown to allow the issue to be pasted into GitHub easily.
+ * Also, provides links with quick access to the githup issue page.
+ */
+public final class ExceptionView extends Alert {
+    private static final String PROJECT_ISSUE_LINK = "https://github.com/WPIRoboticsProjects/GRIP/issues";
```

Comment:
Having this link seems like a good idea.  I could probably be "https://github.com/WPIRoboticsProjects/GRIP/issues/new" to take the user right to the new issue page